Assets/Dev/B/Script/Card.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New Card", menuName = "ScObject/Card")]
+public class Card : ScriptableObject
+{
+    public string skillname;
```

Comment:
Let's use camel casing for our fields and properties. So it would be `skillName` `manaCost` etc